src/app/rpmostree-compose-builtin-tree.cxx

```diff
@@ -438,10 +438,10 @@ install_packages (RpmOstreeTreeComposeContext  *self,
     }
 
   /* Before we install packages, inject /etc/{passwd,group} if configured. */
-  if (!rpmostree_passwd_compose_prep (rootfs_dfd, self->repo, opt_unified_core,
-                                      **self->treefile_rs, self->treefile,
-                                      self->previous_checksum, cancellable, error))
-    return FALSE;
+  g_assert (self->repo);
+  auto previous_ref = self->previous_checksum? : "";
```

Comment:
Not worth a respin but I think this reads more nicely as `self->previous_checksum ?: ""`, thinking of `?:` as the `unwrap_or` operator and not "empty space is default".